pkgs/development/python-modules/python-redis-lock/default.nix

```diff
@@ -12,16 +12,18 @@
   pytestCheckHook,
   pythonOlder,
   redis,
-  withDjango ? false,
   django-redis,
 }:
 
-buildPythonPackage rec {
+let
   pname = "python-redis-lock";
   version = "4.0.0";
+  optional-dependencies.django = [ django-redis ];
```

Comment:
This goes right below dependencies in the function body.

Reply:
I wanted to avoid using `rec` here. Is `optional-dependencies` supposed to be inside `passthru` or is it just an attribute? I see it used differently.

Reply:
It has become a full-blown attribute in b9138b7c072f55dcfd7b8c0747c3c4291bd0b28e

Reply:
Is it okay like this then?